### PR TITLE
[DRAFT][onert] support optional tensor for FULLY_CONNECTED

### PR DESCRIPTION
**Pull request by glistening**
It supports `FULLY_CONNECTED` optional `bias` tensor.

It is necessary to modify loader, core, kernel and so on.

ONE-DCO-1.0-Signed-off-by: Sanggyu Lee <sg5.lee@samsung.com>

### CODE REVIEW

File: runtime/onert/frontend/base_loader/include/base_loader.h [hunk 1]
```diff
@@ -322,16 +330,17 @@ void BaseLoader<LoaderDomain, SpecificLoader>::loadOperationIO(const Operator *o
   {
     // Optional tensors are not supported yet except for FULLY_CONNECTED.
     auto builtin_code = _model->operator_codes()->Get(op->opcode_index())->builtin_code();
-    if (isOptionalInputTensor(idx))
+    if (isOptionalInputTensor(idx) &&
+        builtin_code != BuiltinOperator::BuiltinOperator_FULLY_CONNECTED)
```
Comment (dr-venkman) [inline runtime/onert/frontend/base_loader/include/base_loader.h:334]: Could the condition check for `builtin_code` be expanded to include also other ops that allow for optional input tensor? A list of allowed ops, or a flag for each op indicating use of optional input tensors would help greatly. In future, developers can simply add their `builtin_code` to allow optional input tensors. Do you think this is feasible?

Comment (glistening) [reply runtime/onert/frontend/base_loader/include/base_loader.h:334]: @dr-venkman Thank you for suggestion. Updated.

**Comment by glistening** [comment]
Now, it can run fullyconnected where `bias tensor index = -1`.

[fullyconnected_bias_-1.tflite](https://github.com/Samsung/ONE/issues/820#issuecomment-633350347)

```
$ Product/x86_64-linux.debug/out/bin/nnpackage_run --nnpackage fullyconnected_bias_-1
... run 0 takes 0.02 ms
===================================
MODEL_LOAD takes 3.119 ms
PREPARE    takes 16.185 ms
EXECUTE    takes 
- Min:  0.02 ms
- Max:  0.02 ms
- Mean: 0.02 ms
- GeoMean: 0.02 ms
===================================
```

File: runtime/onert/frontend/base_loader/include/base_loader.h [hunk 2]
```diff
@@ -333,12 +333,17 @@ void BaseLoader<LoaderDomain, SpecificLoader>::loadOperationIO(const Operator *o
...
+      auto builtin_code = _model->operator_codes()->Get(op->opcode_index())->builtin_code();
+      std::vector<BuiltinOperator> allowed = {BuiltinOperator::BuiltinOperator_FULLY_CONNECTED};
+      if (isOptionalInputTensor(idx) &&
+          std::find(allowed.begin(), allowed.end(), builtin_code) == allowed.end())
+        throw std::runtime_error(
+            std::string("loader doesn't support optional input tensor yet for ")
+                .append(EnumNameBuiltinOperator(builtin_code)));
+    };
+    check_optional_input();
+    inputs.append(tensorIdxToOperandIdx(idx));
```
Comment (glistening) [inline runtime/onert/frontend/base_loader/include/base_loader.h:346]: I accidentally pushed this changes in this PR. I already have pushed this change in #1602. Anyway, I will close #1602 if this PR is merged with this change.

Comment (glistening) [reply runtime/onert/frontend/base_loader/include/base_loader.h:346]: @hseok-oh Oops... I've mistaken PR. It is draft PR. It is not for merging. I apologize for confusion. I will add `[DRAFT]` in title again, `DRAFT` label is not enough.

**Comment by hseok-oh** [comment]
@glistening I've changed this PR to draft

**Pull request closed without merging**